components/docs-chef-io/content/automate/reusable/md/restore_troubleshooting.md

```diff
@@ -1,31 +1,33 @@
-Try these steps if Chef Automate returns an error while restoring data.
+Follow the steps listed below if Chef Automate encounters an error during data restoration.
 
 1. Check the Chef Automate status.
 
    ```sh
    chef-automate status
    ```
 
-1. Check the status of your Habitat service on the Automate node.
+2. Check the status of your Habitat service on the Automate node.
 
    ```sh
    hab svc status
    ```
 
-1. If the deployment services are not healthy, reload them.
+3. If the deployment services are not healthy, reload them.
 
     ```sh
     hab svc load chef/deployment-service
     ```
 
-Now check the status of the Automate node and then try running the restore command from the bastion host.
+4. Check the status of the Automate node, and then attempt to run the restore command from the bastion host.
 
-1. How to change the `base_path` or `path`. The steps for the File System backup are as shown below:
+5. To change the `base_path` or `path`, follow the steps below for performing a backup.
 
-   - While at the time of deployment `backup_mount` default value will be `/mnt/automate_backups`
-   - In case, if you modify the `backup_mount` in `config.toml` before deployment, then the deployment process will do the configuration with the updated value
-   - In case, you changed the `backup_mount` value post-deployment, then we need to patch the configuration manually to all the frontend and backend nodes, for example, if you change the `backup_mount` to `/bkp/backps`
-   - Update the FE nodes with the below template, use the command  `chef-automate config patch fe.toml --fe`
+   * File System
+
+      * By default, the `backup_mount` is set to `/mnt/automate_backups` during deployment.
+      * If you update the `backup_mount` value in the `config.toml` file before deployment, the deployment process will automatically apply the updated path.
+      * If the `backup_mount` value is changed after deployment (e.g., to /bkp/backps), you must manually patch the configuration on all frontend and backend nodes.
```

Comment:
Replace it with: * If the `backup_mount` value changes after deployment (e.g., to /bkp/backps), you must manually patch the configuration on all frontend and backend nodes.